src/commands/utils/ping.ts

```diff
@@ -6,7 +6,7 @@ export const ping: CommandDefinition = {
     name: 'ping',
     description: 'Send back a message',
     category: CommandCategory.UTILS,
-    requiredPermissions: ['ManageWebhooks'],
+    requirements: { permissions: ['ManageWebhooks'] },
```

Comment:
Not a comment on the PR, but @benw202 : Does this make sense, should we take the opportunity to change this to BOT_DEVELOPER, ADMIN & MODERATOR roles?